.gitignore

```diff
@@ -97,60 +175,6 @@ $RECYCLE.BIN/
 # Windows shortcuts
 *.lnk
 
-
-### Eclipse ###
-
-.metadata
-bin/
-tmp/
-*.tmp
-*.bak
-*.swp
-*~.nib
-local.properties
-.settings/
-.loadpath
-.recommenders
-
-# Eclipse Core
-.project
-
-# External tool builders
-.externalToolBuilders/
-
-# Locally stored "Eclipse launch configurations"
-*.launch
-
-# PyDev specific (Python IDE for Eclipse)
-*.pydevproject
-
-# CDT-specific (C/C++ Development Tooling)
-.cproject
-
-# JDT-specific (Eclipse Java Development Tools)
-.classpath
-
-# Java annotation processor (APT)
-.factorypath
-
-# PDT-specific (PHP Development Tools)
-.buildpath
-
-# sbteclipse plugin
-.target
-
-# Tern plugin
-.tern-project
-
-# TeXlipse plugin
-.texlipse
-
-# STS (Spring Tool Suite)
-.springBeans
-
-# Code Recommenders
-.recommenders/
-
-### Project-specific ###
-# Ignore Docker environment variable files
-**/*.env
\ No newline at end of file
+### TDS Additions ###
```

Comment:
The defaults created by gitignore.io capture most of the files but there are a few IntelliJ files that are missed.  Instead of sprinkling them throughout the file I created a `TDS Additions` section.  This will make these easier to find when looking across projects.